src/Servers/Kestrel/Transport.Quic/test/WebHostTests.cs

```diff
@@ -122,7 +142,7 @@ public async Task Listen_Http3AndSocketsCoexistOnSameEndpoint_ClientSuccess()
                 webHostBuilder
                     .UseKestrel(o =>
                     {
-                        o.Listen(IPAddress.Parse("127.0.0.1"), 5005, listenOptions =>
+                        o.Listen(IPAddress.Parse("127.0.0.1"), 0, listenOptions =>
```

Comment:
Nit: I know other tests make use of this, but should we have a test specifically verifying port-0 binding works when just HTTP/3 is enabled?

Reply:
I don't think it's necessary. Lots of tests rely on it working.